### ISSUE TITLE
Database update

### ISSUE THREAD

**Issue by blaizereal** [jiarong/VirSorter2#191]
Dear jiarong,

Will you be able to update the database with the latest PFAM/HMM files? If not, is there any way to do it in-house? 

Thanks: blaize

**Comment by jiarong** [comment]
Hi blaize, I am planning to do an overall database update but quite overwhelmed with other things now, so likely in summer. If you want to update Pfam on your own, you just need to update the following files in `db/hmm/pfam/`:
```
Pfam-A-Archaea.hmm
Pfam-A-Bacteria.hmm
Pfam-A-Eukaryota.hmm
Pfam-A-Mixed.hmm
Pfam-A-Viruses.hmm
```
The rule to define host domain is if >=90% seqs in a Pfam then it is that domain specific.  Otherwise, it's "Mixed". Pfam seqs are from UniProt and you can find their taxonomy there.

**Comment by SergeyBaikal** [comment]
Dear [jiarong](https://github.com/jiarong)
I join the question and am also waiting for the database to be updated.

**Comment by SergeyBaikal** [comment]
> Pfam-A-Archaea.hmm
> Pfam-A-Bacteria.hmm
> Pfam-A-Eukaryota.hmm
> Pfam-A-Mixed.hmm
> Pfam-A-Viruses.hmm

Wouldn't it be easier to implement automatic updating on request with some flag or as a separate script for database generation in future versions? I don't understand — do I simply need to replace these files:
Pfam-A-Archaea.hmm
Pfam-A-Bacteria.hmm
Pfam-A-Eukaryota.hmm
Pfam-A-Mixed.hmm
Pfam-A-Viruses.hmm? Or is something else required? Please clarify.

**Comment by jiarong** [comment]
Right, that's what's needed to update the Pfam component of the DB.

**Comment by SergeyBaikal** [comment]
Sorry, but where should I look for these files?  `https://ftp.ebi.ac.uk/pub/databases/Pfam/` 

I see the following directory structure
db ----conda_envs
     ---- group
     ---- hmm
     ---- rbs     
How then will the groups be updated if we only update the files from the hmm directory?
